build/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN make go-build
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10-1179.1739286367
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1742914212
```

Comment:
Did make validate set this to ubi9? I think we're still using ubi8 on most if not all operators

Reply:
We had issues with dynamic linking I think when trying to go to ubi9. For example, the boilerplate image is still ubi8

Reply:
ACK, updating boilerplate replaces that back to 8 again.  9 _might_ be required for a libxml2 CVE patch, and I was basing it on another non-boilerplate update.

I didn't realize boilerplate would revert this, so we'll go with this and validate the libxml2 after.

Reply:
Yeah, confirmed `libxml2` is patched in the latest UBI8 image from boilerplate, so we're good to go:

```
[chcollin@toolbx route-monitor-operator]$ podman run -it registry.access.redhat.com/ubi8/ubi-minimal:8.10-1179.1741863533 rpm -q libxml2
libxml2-2.9.7-19.el8_10.x86_64

```